scripts/thanks.js

```diff
@@ -0,0 +1,33 @@
+if (!remoteUrl) {
+  var remoteUrl = "https://api.carpoolvote.com/live";
+}
+
+const handleBackendResponse = () => {
```

Comment:
I've looked into adding a transpilation stage but there's a strict list of jekyll plugins that are whitelisted on Github Pages, and the Babel one isn't among them. So currently we're limited to what Github Pages supports. We could swap for something else but we'd need a build step before deploying, and I think that would add unnecessary overhead.

Good point the [contrib doc](https://github.com/voteamerica/voteamerica.github.io/blob/master/docs/contributing.md): I've updated it to specifically mention this.